cla_public/templates/checker/result/face-to-face.html

```diff
@@ -67,9 +67,8 @@ <h2 class="govuk-heading-m">{{ _('What happens next') }}</h2>
     </p>
 
     <p class="govuk-body">
-        <a class="govuk-link" href="https://www.gov.uk/done/check-if-civil-legal-advice-can-help-you" aria-labelledby="aria-satisfaction-survey">
-          {% trans %}What did you think of this service?{% endtrans %}
-        </a> ({% trans %}takes 30 seconds{% endtrans %})
+        {% trans %}<a class="govuk-link" href="https://www.gov.uk/done/check-if-civil-legal-advice-can-help-you" aria-labelledby="aria-satisfaction-survey">
+          What did you think of this service?</a> (takes 30 seconds){% endtrans %}
```

Comment:
Can this be
`<a class="govuk-link" href="https://www.gov.uk/done/check-if-civil-legal-advice-can-help-you" aria-labelledby="aria-satisfaction-survey">{% trans %}What did you think of this service?{% endtrans %}</a> {% trans %}(takes 30 seconds){% endtrans %}`